src/App/App.js

```diff
@@ -94,7 +125,12 @@ class App extends React.Component {
       art: { records: [...this.state.art.records, ...data.records]},
       isLoading: false,
     }))
-    .catch(err => console.error(err))
+    .catch(err => {console.error(err)
+      this.setState({
+        isLoading: false,
+        error: err
+      })
```

Comment:
Still amazed I didn't realize this was why errors were not happening until today 😂